### DISCUSSION TITLE
add traefik and dokploy containers as project

### DISCUSSION THREAD

**Discussion by Manubi** [Dokploy/dokploy#76] [Ideas]
Hi,
imho it would be nice to see the dokploy and traefik containers inside the dokploy dashboard. Just to see an overview of what's running in containers and what the settings they have. Makes it easier to get going for beginners as they can just try to copy settings and it would be easier to reason. Also an overview of all the containers running and the port settings would be cool.
Thanks

**Comment by Siumauricio** [comment]
Basically you can see this by accessing server-ip:8080
